design-documents/graph-ql/coverage/catalog/configurable-options-selection.graphqls

```diff
@@ -0,0 +1,21 @@
+type ConfigurableProduct {
+    configurable_options_selection_metadata(selectedConfigurableOptionValues: [ID!]): ConfigurableOptionsSelectionMetadata @doc(description: "Metadata for the specified configurable options selection")
```

Comment:
It was pretty hard to determine what the API’s should do. The naming didn’t make it clear what the `configurable_options_selection_metadata` will return.

It seems the return value of `configurable_options_selection_metadata` is some form of aggregate of the remaining possible simples?

`metadata` is a very generic word? Maybe omit that? `configurable_options_selection`

Reply:
Paul, thanks for reviewing. We can extend description of the field. Please check https://github.com/magento/architecture/pull/394/files?short_path=567352f#diff-567352fc8df3d0073eac2ea540e6f57f and let me know what would you like to include in the `@doc`. I think that document also answers your questions regarding extensibility and price range.

Reply:
Also, regarding `metadata` in the name. Selection is provided as an input, so we are getting metadata for the provided selection.

Reply:
> Selection is provided as an input, so we are getting metadata for the provided selection.

Yes, that is true that it returns some form of metadata, but following that logic you can also call it: `configurable_options_selection_metadata_data_aggregate` Which still is true, but doesn't add any value?

In my opinion the word `metadata` doesn't add any value so it can be omitted.